<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFRCScouter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="team">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Your Team:           </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="backButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="nextButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>12</verstretch>
       </sizepolicy>
      </property>
      <widget class="WelcomePage" name="welcome"/>
      <widget class="SubjectiveScales" name="scales"/>
      <widget class="Notes" name="notes"/>
      <widget class="QRCode" name="qrcode"/>
      <widget class="TeamInfo" name="teamInfo"/>
      <widget class="MatchPage" name="autoScouting"/>
      <widget class="MatchPage" name="teleScouting"/>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="station">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Red 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WelcomePage</class>
   <extends>QWidget</extends>
   <header>WelcomePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TeamInfo</class>
   <extends>QWidget</extends>
   <header>TeamInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QRCode</class>
   <extends>QWidget</extends>
   <header>QRCode.h</header>
   <container>1</container>
   <slots>
    <signal>backButtonPressed()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>Notes</class>
   <extends>QWidget</extends>
   <header>Notes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubjectiveScales</class>
   <extends>QWidget</extends>
   <header>SubjectiveScales.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatchPage</class>
   <extends>QWidget</extends>
   <header>MatchPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>next()</slot>
  <slot>back()</slot>
  <slot>backToStart()</slot>
 </slots>
</ui>
